src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -386,64 +412,68 @@ var eventTypes = {
+  topAbort:               eventTypes.abort,
+  topAnimationEnd:        eventTypes.animationEnd,
+  topAnimationIteration:  eventTypes.animationIteration,
+  topAnimationStart:      eventTypes.animationStart,
+  topBlur:                eventTypes.blur,
+  topCanPlay:             eventTypes.canPlay,
+  topCanPlayThrough:      eventTypes.canPlayThrough,
+  topClick:               eventTypes.click,
+  topContextMenu:         eventTypes.contextMenu,
+  topCopy:                eventTypes.copy,
+  topCut:                 eventTypes.cut,
+  topDoubleClick:         eventTypes.doubleClick,
+  topDrag:                eventTypes.drag,
+  topDragEnd:             eventTypes.dragEnd,
+  topDragEnter:           eventTypes.dragEnter,
+  topDragExit:            eventTypes.dragExit,
+  topDragLeave:           eventTypes.dragLeave,
+  topDragOver:            eventTypes.dragOver,
+  topDragStart:           eventTypes.dragStart,
+  topDrop:                eventTypes.drop,
+  topDurationChange:      eventTypes.durationChange,
+  topEmptied:             eventTypes.emptied,
+  topEncrypted:           eventTypes.encrypted,
+  topEnded:               eventTypes.ended,
+  topError:               eventTypes.error,
+  topFocus:               eventTypes.focus,
+  topInput:               eventTypes.input,
+  topInvalid:             eventTypes.invalid,
+  topKeyDown:             eventTypes.keyDown,
+  topKeyPress:            eventTypes.keyPress,
+  topKeyUp:               eventTypes.keyUp,
+  topLoad:                eventTypes.load,
+  topLoadedData:          eventTypes.loadedData,
+  topLoadedMetadata:      eventTypes.loadedMetadata,
+  topLoadStart:           eventTypes.loadStart,
+  topMouseDown:           eventTypes.mouseDown,
+  topMouseMove:           eventTypes.mouseMove,
+  topMouseOut:            eventTypes.mouseOut,
+  topMouseOver:           eventTypes.mouseOver,
+  topMouseUp:             eventTypes.mouseUp,
+  topPaste:               eventTypes.paste,
+  topPause:               eventTypes.pause,
+  topPlay:                eventTypes.play,
+  topPlaying:             eventTypes.playing,
+  topProgress:            eventTypes.progress,
+  topRateChange:          eventTypes.rateChange,
+  topReset:               eventTypes.reset,
+  topScroll:              eventTypes.scroll,
+  topSeeked:              eventTypes.seeked,
+  topSeeking:             eventTypes.seeking,
+  topStalled:             eventTypes.stalled,
+  topSubmit:              eventTypes.submit,
+  topSuspend:             eventTypes.suspend,
+  topTimeUpdate:          eventTypes.timeUpdate,
+  topTouchCancel:         eventTypes.touchCancel,
+  topTouchEnd:            eventTypes.touchEnd,
+  topTouchMove:           eventTypes.touchMove,
+  topTouchStart:          eventTypes.touchStart,
+  topTransitionEnd:       eventTypes.transitionEnd,
+  topVolumeChange:        eventTypes.volumeChange,
+  topWaiting:             eventTypes.waiting,
+  topWheel:               eventTypes.wheel,
```

Comment:
Can you actually just add your items and not re-format the whole block. We should fix the formatting separately (and get rid of the whitespace)